Parse/Internal/File/Controller/PFFileStagingController.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import <Bolts/BFTask.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol PFFileManagerProvider;
```

Comment:
Supernit: You don't really need this to be inside NS_ASSUME_NONNULL, but it's ok.